es-design-system/pages/molecules/es-form-msg.vue

```diff
@@ -8,17 +8,42 @@
                 bootstrap-vue alert
             </b-link>
         </p>
-        <es-button @click="fakeFormMsg()">
-            Show Success
+
+        <es-button
+            @click="showSuccess = true">
+            Show success
+        </es-button>
+        <es-button
+            @click="showError = true">
+            Show error
         </es-button>
-        <es-button @click="fakeFormMsg(false)">
-            Show Error
+        <es-button
+            @click="showInfo = true">
+            Show info
         </es-button>
+
+        <es-form-msg
+            class="my-450"
+            :show="showSuccess"
+            :variant="success"
+            @hidden="showSuccess = false">
+            Saved successfully
+        </es-form-msg>
         <es-form-msg
             class="my-450"
-            :variant="formMsgVariant"
-            :message="formMsg"
-            @hidden="formMsg = ''" />
+            :show="showError"
+            :variant="danger"
```

Comment:
Same thing here - we need to remove the colon in front so this is passed in as a string rather than a nonexistent variable.